### DISCUSSION TITLE
How to: Allow Administrators or Global Mods in your WikiGroupName

### DISCUSSION THREAD

**Discussion by Crypto2** [Digitalroot/MediaWiki_PHPBB_Auth#63] [Q&A]
To allow those groups to log in you have to use: ADMINISTRATORS and GLOBAL_MODERATORS instead of their pretty names. I spent longer than I'd like to admit figuring that out lol.

Also if your forum is on the same DB server but in a different DB you can use this format to not have to open a 2nd connection:

```
dbname`.`phpbb_users
```
(the query in the plugin already has the outer backticks)

**Comment by cpeel** [comment]
Thanks for the info!

> Also if your forum is on the same DB server but in a different DB you can use this format to not have to open a 2nd connection

I assume that requires both databases use the same creds? That is, the wiki's database user has at least SELECT access to the phpBB database & tables?

**Comment by Crypto2** [comment]
Correct
